echo-telemetry/src/main/java/com/netflix/spinnaker/echo/config/TelemetryConfig.java

```diff
@@ -75,7 +78,24 @@ private OkClient telemetryOkClient(TelemetryConfigProps configProps) {
     String endpoint = DEFAULT_TELEMETRY_ENDPOINT;
     String instanceId = new ULID().nextULID();
     String spinnakerVersion = "unknown";
+    DeploymentMethod deploymentMethod = new DeploymentMethod();
     int connectionTimeoutMillis = 3000;
     int readTimeoutMillis = 5000;
+
+    @Data
+    @NoArgsConstructor
+    @AllArgsConstructor
```

Comment:
Do you need all of these? By my eyeball, it looks like you may only need `Data`, which has `NoArgsConstructor` when `AllArgsConstructor` is missing. IIRC Spring uses each fields setter method when deserializing the config, so AllArgsConstructor may not be necessary.